components/brave_wallet_ui/components/desktop/views/portfolio/portfolio-asset.tsx

```diff
@@ -539,19 +558,19 @@ export const PortfolioAsset = (props: Props) => {
 
         <LineChart
           priceData={
-            selectedAsset
-              ? formattedPriceHistory
+           selectedAssetFromParams 
+              ? formattedPriceHistory || []
```

Comment:
perf nit: let's define this empty array outside of the component so the reference doesn't change/prevent re-renders

```suggestion
              ? formattedPriceHistory || emptyPriceList
```